docker-compose.yml

```diff
@@ -8,6 +8,17 @@ services:
       SA_PASSWORD: Th3PA55--8zz
     volumes:
       - leaf-mssql:/var/opt/mssql
+      - ./src/db/build/docker-entrypoint.sh:/usr/local/bin/docker-entrypoint.sh:ro
+      - ./src/db/build/configure-db.sh:/usr/local/bin/configure-db.sh:ro
+      - ./src/db/build/setup.sql:/opt/mssql/setup.sql:ro
+
+      - ./src/db/build/LeafDB.Schema.sql:/docker-entrypoint-initdb.d/00LeafDB.Schema.sql:ro
```

Comment:
Maybe put this in `/usr/share/docker-entrypoint/` instead? Mostly an aesthetic thing, not a blocker. That seems like a reasonable place to put things like this, but OTOH it is a bind mount from outside the container so maybe it should be marked as special in this way

Reply:
I agree- a random directory off root is not a great place, since it goes against all existing standards for linux filesystems (eg [FHS](https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard)).

...however, this functionality was copied from [the official postgres docker image, which uses the same directory](https://github.com/docker-library/docs/blob/master/postgres/README.md#initialization-scripts). 

I'm hesitant to change it, but will give it some more thought and see if there's a way to make it both configurable, and obvious